src/c++/vdo/base/dm-vdo-target.c

```diff
@@ -2981,6 +2989,18 @@ static int vdo_preresume_registered(struct dm_target *ti, struct vdo *vdo)
 		vdo_log_info("starting device '%s'", device_name);
 		result = perform_admin_operation(vdo, LOAD_PHASE_START, load_callback,
 						 handle_load_error, "load");
+		if (result == VDO_UNSUPPORTED_VERSION) {
+			/*
+			 * A component version is not supported and the load has been aborted.
+			 */
+			vdo_log_error("Start failed, suspending the VDO");
```

Comment:
A minor thiing, I think "suspending VDO" is better than "suspending _the_ VDO". I'm also not sure we need to log anything here given we are generating a log message at every level of the stack for this case.

Reply:
I added it for consistency in the code. I can remove it.